feed-integration-webapp/src/main/java/org/bahmni/module/feedintegration/apicalls/CraterAPIcalls.java

```diff
@@ -0,0 +1,238 @@
+package org.bahmni.module.feedintegration.apicalls;
+
+import org.apache.http.*;
+import org.apache.http.client.methods.*;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.bahmni.module.CraterAtomfeed;
+import org.bahmni.module.feedintegration.atomfeed.contract.patient.OpenMRSPatientFullRepresentation;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import org.springframework.context.annotation.Configuration;
+
+import org.springframework.stereotype.Component;
+
+import java.io.*;
+import java.net.*;
+
+import java.util.Properties;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+
+@Configuration
+@Component
+public class CraterAPIcalls {
+
+    String auth = CraterAtomfeed.auth;
+
+    public Properties getprop() throws IOException{
```

Comment:
properties could be a singleton dependency or a private construct 

```
public CraterAPIcalls() throws IOException {
        auth = CraterAtomfeed.auth;
        properties = getProps();
    }
```
and then the `CraterAPIcalls` class would have `Properties properties;`